.github/scripts/findUnusedStyles.ts

```diff
@@ -0,0 +1,462 @@
+                        }
+                    }
+                    return; // Don't continue traversing for styles function
+                }
+
+                // Skip processing top-level constants and other functions for styles/index.ts
+                if (ts.isVariableDeclaration(node) || ts.isFunctionDeclaration(node)) {
+                    return;
+                }
+            } else {
+                // For other files, process all style-like nodes normally
+                this.processStyleNode(node, sourceFile, file);
+            }
+
+            ts.forEachChild(node, visit);
+        };
+
+        visit(sourceFile);
+    }
+
+    private processObjectProperties(objectLiteral: ts.ObjectLiteralExpression, sourceFile: ts.SourceFile, file: string): void {
+        for (const property of objectLiteral.properties) {
+            if (ts.isPropertyAssignment(property) || ts.isMethodDeclaration(property)) {
+                const key = this.extractKeyFromPropertyNode(property);
+
+                if (key && this.isStyleKey(key)) {
+                    try {
+                        let shouldCapture = false;
+                        if (ts.isPropertyAssignment(property)) {
+                            // For PropertyAssignment, check if the value is an object literal
+                            if (ts.isObjectLiteralExpression(property.initializer)) {
+                                shouldCapture = true;
+                            }
+                        } else if (ts.isMethodDeclaration(property)) {
+                            // For MethodDeclaration, it's a function that returns styles
+                            shouldCapture = true;
+                        }
+
+                        if (shouldCapture) {
+                            const {line, character} = sourceFile.getLineAndCharacterOfPosition(property.getStart());
+                            this.styleDefinitions.set(key, {
+                                key,
+                                file,
+                                line: line + 1,
+                                column: character + 1,
+                            });
+                        }
+                    } catch (error) {
+                        console.warn(`Warning: Could not get position for style '${key}' in ${file}`);
+                    }
+                }
+            }
+        }
+    }
+
+    private extractKeyFromPropertyNode(node: ts.PropertyAssignment | ts.MethodDeclaration): string | undefined {
+        if (ts.isPropertyAssignment(node)) {
+            if (ts.isIdentifier(node.name) || ts.isStringLiteral(node.name)) {
+                return node.name.text;
+            }
+        } else if (ts.isMethodDeclaration(node) && ts.isIdentifier(node.name)) {
+            return node.name.text;
+        }
+        return undefined;
+    }
+
+    private processStyleNode(node: ts.Node, sourceFile: ts.SourceFile, file: string): void {
```

Comment:
Similar to above, `processStyleNode` isn't super clear what it does. Let's improve the name and/or add a JSDoc explaining what it does.